build-logic/integration-testing/src/main/kotlin/gradlebuild/integrationtests/shared-configuration.kt

```diff
@@ -126,6 +131,10 @@ fun Project.getBucketProvider() = gradle.sharedServices.registerIfAbsent("buildB
     parameters.repoRoot.set(repoRoot())
 }
 
+internal
+class SamplesBaseDirPropertyProvider(@InputFiles @PathSensitive(PathSensitivity.RELATIVE) val autoTestedSamplesPath: Directory) : CommandLineArgumentProvider {
```

Comment:
```suggestion
class SamplesBaseDirPropertyProvider(@InputDirectory @PathSensitive(PathSensitivity.RELATIVE) val autoTestedSamplesPath: Directory) : CommandLineArgumentProvider {
```

Actually, you can use `@InputDirectory` here, given that it will be an existing directory.